packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -413,6 +413,111 @@ input:-webkit-autofill {
   outline: none;
   box-shadow: 0px 0px 0px 1px var(--notifi-focus-color);
 }
+.NotifiSmsInput__dropdownSelected {
+  display: flex;
+  align-items: center;
+  width: 100%;
```

Comment:
instead of width: 100%; and height: 100%,
can we add `display: flex` to the parent property and make this flex: 1? so it can take up the container?